packages/office-ui-fabric-react/src/components/Link/__snapshots__/Link.test.tsx.snap

```diff
@@ -15,7 +15,11 @@ exports[`Link renders Link correctly 1`] = `
         text-decoration: none;
       }
       .ms-Fabric--isFocusVisible &:focus {
-        outline: 1px solid #666666;
+        box-shadow: 0 0 0 1px #666666 inset;
+        outline: 1px solid #ffffff;
```

Comment:
any reason why this has an outline

Reply:
(i'm not sure where this is coming from.

Reply:
ooh thanks for catching this! must've been left over from when I was trying out a bunch of different solutions

Reply:
This is weird - I tried to clear all instances of this out but when I run "npm run update-snapshots", they're all added back in... I think it's because I added outline: `1px solid ${theme.palette.white}` to the selector for high contrast and it evaluates to `1 px solid #ffffff`

Reply:
i'm still really confused here... I don't see where this could be coming from. I see in the change you made, you replaced outline with boxshadow. So we should have seen the same replacement in shapshots, but instead there's an addition.

Reply:
I've tried a bunch of different things to figure out why this is added and I'm really not sure - when I changed the outline from theme.palette.white to WindowText, this got changed from #ffffff to WindowText so for some reason, the high contrast selector rule is being added under the general focus styling. And I can't remove the selector for focus under the high contrast because when I do that, the outline's always there in HC mode.